File: API/Schemas/sell.py
from pydantic import BaseModel 
from decimal import Decimal
from datetime import datetime
class Sell(BaseModel):
    idVenta:int | None = None
    numeroVenta:str
    idCliente:int
    subTotal:Decimal
    impuestoTotal:Decimal
    Total:Decimal
    fechaRegistro:datetime
    class Config:
        orm_mode = True
        allow_population_by_field_name = True
        arbitrary_types_allowed = True

File: Frontend/src/app/models/sell.ts
import { Detail } from "./detail";

export interface Sell {
    idVenta?:number,
    numeroVenta:any,
    idCliente:number,
    subTotal:number,
    impuestoTotal:number,
    Total:number,
    fechaRegistro:any,
    nombre:string,
    cedula:string,
    detalles?:Detail[]
    telefono:string
}
